tests/webschema.validator.tests.ps1

```diff
@@ -0,0 +1,42 @@
+$Here = ''
+if ($MyInvocation.MyCommand.Path) {
+    $Here = Split-Path $MyInvocation.MyCommand.Path
+} else {
+    $Here = $PWD -replace '^\S+::',''
+}
+
+Add-Type -Path "$Here\..\nancy\Nancy.dll"
+Add-Type -Path "$Here\..\nancy\Nancy.Hosting.Self.dll"
+Invoke-Expression (Get-Content -Path "$Here\..\flancy.psm1" | Out-String)
+
+$Empty = $null
+$NotADictionary = 1,2,3
+$EmptyProperty = @{Path = '/' ; Script = {"Hello World!"}}
+$MultipleProperties = @{Path = '/';Method = 'Foo','Get' ; Script = {"Hello World!"}}
+$InvalidPath = @{Path = '\\\' ; Method = 'Get' ; Script = {"Hello World!"}}
+$InvalidMethod = @{Path = '/' ; Method = 'Foo' ; Script = {"Hello World!"}}
+$InvalidScript = @{Path = '/' ; Method = 'Get' ; Script = '~!@#$%^&*()_+'}
+
+Describe "Flancy web schema validator" {
+    It "Should throw an error when calling New-Flancy with the custom web schema, when it's empty" {
```

Comment:
I'm going to modify this - Rather than put the vars up top and have to scroll back and forth, it's better to put it in the It {} block.